pkg/provider/provider.go

```diff
@@ -205,7 +208,6 @@ func (k *kubeProvider) Check(ctx context.Context, req *pulumirpc.CheckRequest) (
 		}
 	}
 
-	// We currently don't change the inputs during check.
 	return &pulumirpc.CheckResponse{Inputs: newResInputs, Failures: failures}, nil
```

Comment:
will `newResInputs` contain the autoname, then?

Reply:
It will, yes.